Ginger/Ginger/Ginger.csproj

```diff
@@ -342,6 +342,7 @@
       <Private>True</Private>
       <Private>True</Private>
     </Reference>
+    <Reference Include="System.ServiceModel" />
```

Comment:
need to check what are those changes for and if we need to merge them to master

Reply:
Removed the unnecessary/accidentally added refs.